baseplate/experiments/providers/variant_sets/multi_variant_set.py

```diff
@@ -0,0 +1,52 @@
+from .base import VariantSet
+
+
+class MultiVariantSet(VariantSet):
+
+    def __init__(self, variants, num_buckets=1000):
+        self._validate_variants(variants)
+        self.variants = variants
+        self.num_buckets = num_buckets
+
+    def __contains__(self, item):
+        for variant in self.variants:
+            if variant.get('name') == item:
+                return True
+
+        return False
+
+    def _validate_variants(self, variants):
+
+        if variants is None:
+            raise ValueError('No variants provided')
+
+        if len(variants) < 3:
+            raise ValueError("MultiVariant experiments expect two controls "
```

Comment:
Should this also check that at least 2 of the variants are named `control_X`?

Reply:
I don't think the system should enforce variant naming.

Reply:
Makes sense, should the error message not reference "control" then?  I feel like that implies that it is looking for variants called "control".

Reply:
Makes sense. Changed.